src/io/mod.rs

```diff
@@ -51,10 +51,14 @@ use flate2::bufread::MultiGzDecoder;
 use flate2::write::GzEncoder;
 use flate2::Compression;
 use serde::{de::DeserializeOwned, Serialize};
+use zstd::stream::{Decoder, Encoder};
```

Comment:
nit: import as ZstdDecoder and ZstEncoder?

Reply:
Much better now